src/curated-corpus/components/CustomSectionDetails/CustomSectionDetails.tsx

```diff
@@ -313,6 +332,33 @@ export const CustomSectionDetails: React.FC<CustomSectionDetailsProps> = ({
             </IconButton>
             <IconButton
               size="small"
+              onClick={async () => {
+                if (
+                  window.confirm(
```

Comment:
lets not use window.confirm but instead, use our custom modal like we do in other parts of the tool.